docsite/docs/Docs/fal-cli/fal-el-jobs.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 6
+---
+
+# Run Extract-Load jobs
+`fal` makes it easy to run EL jobs on your data pipelines. Two common types of EL pipeline providers are supported: [Airbyte](https://docs.airbyte.com/) and [Fivetran](https://fivetran.com/docs/getting-started). More providers will be available in near future.
+
+In order to be run EL jobs, you first need to setup EL configurations in [`profiles.yml`](../credentials-profile#exctract-load-configuration).
+
+Here's how you trigger an Airbyte job:
```

Comment:
How about skipping this whole section of `el.airbyte_sync...` and `el.fivetran_sync...` and just directing users to the functions?

Reply:
I think it's good to see an example here without navigating to a different page